WORKSPACE

```diff
@@ -118,13 +118,8 @@ maven_install(
         "junit:junit:4.12",
         "com.google.truth:truth:0.43",
         "com.github.bumptech.glide:glide:4.11.0",
-        "com.github.bumptech.glide:compiler:4.11.0",
```

Comment:
If these deps weren't needed, why were they added originally? Can we remove them from upstream PRs?

Reply:
On it